src/Command/AllCommand.php

```diff
@@ -27,6 +27,16 @@
  */
 class AllCommand extends BakeCommand
 {
+    /**
+     * All commands to call.
+     *
+     * @var string[]
+     */
+    protected $commands = [
+        'Bake\Command\ModelCommand',
```

Comment:
Add `use` statement for each at top of class then then use `ModelCommand::class` instead of `'Bake\Command\ModelCommand'`.